test/src/core/SceneRunnerCSharpSceneTest.cs

```diff
@@ -71,6 +71,17 @@ public void LoadSceneBinaryFormat()
             .IsNotNull();
     }
 
+
+    [TestCase]
+    public void LoadSceneNode()
+    {
+        var currentScene = new Node();
```

Comment:
The test still needs to be improved.
Please use a real scene to load instead of a simple node to reflelct the use case of the issue.
- load a scene 
- do instance it
- do call the custom init method
- do load the scene by the scene runner
- check is scene the same instance 
- check the class variables has still the values of the custom init call